README.md

```diff
@@ -1,24 +1,101 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Recipe-App
```

Comment:
- [ ] Nice one adding a descriptive README. However, kindly add instructions on how to `create` and `migrate` your database so that users can easily find it in the README.

Reply:
Done